### DISCUSSION TITLE
Extensions. VS 2022 or VS 2019 for SSMS

### DISCUSSION THREAD

**Discussion by OilUnion** [VsixCommunity/Community.VisualStudio.Toolkit#473] [Q&A] [answered]
How to write extensions for SSMS on VS (2019 or 2022)? Now only VS 2017 works out of the box.

**Comment by ErikEJ** [comment, marked as answer]
SSMS does not run under the VS 2019 or 2022 Shell, so not possible

**Reply by OilUnion** [reply]
Thanks
